# Description: the default sound theme is "ubuntu" for us
# UbuntuSpecific: yes
#
Index: gnome-control-center-3.3.90/panels/sound/gvc-sound-theme-chooser.c
===================================================================
--- gnome-control-center-3.3.90.orig/panels/sound/gvc-sound-theme-chooser.c	2012-02-20 11:22:59.000000000 +0100
+++ gnome-control-center-3.3.90/panels/sound/gvc-sound-theme-chooser.c	2012-02-21 13:39:53.439109476 +0100
@@ -67,7 +67,7 @@
 #define DEFAULT_ALERT_ID        "__default"
 #define CUSTOM_THEME_NAME       "__custom"
 #define NO_SOUNDS_THEME_NAME    "__no_sounds"
-#define DEFAULT_THEME           "freedesktop"
+#define DEFAULT_THEME           "ubuntu"
 
 enum {
         THEME_DISPLAY_COL,
